### PR TITLE
ali first caption

### PR DESCRIPTION
**Pull request by alireamandegar**

### CODE REVIEW

File: _posts/0000-01-07-alireamandegar.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Ali Caption
```
Comment (beardofedu) [inline _posts/0000-01-07-alireamandegar.md:6]: Nice addition! 👍

File: _posts/0000-01-07-alireamandegar.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (beardofedu) [inline _posts/0000-01-07-alireamandegar.md:3]:
```suggestion
title: "Welcome to alireamandegar's slide!"
```

**Comment by DelphiDesigner** [comment]
Hi Ali.

**Comment by alireamandegar** [comment]
Rebased & Merged

**Pull request closed without merging**